src/main/java/io/reactivex/internal/operators/single/SingleFromCallable.java

```diff
@@ -28,20 +32,29 @@ public SingleFromCallable(Callable<? extends T> callable) {
     }
 
     @Override
-    protected void subscribeActual(SingleObserver<? super T> s) {
+    protected void subscribeActual(SingleObserver<? super T> observer) {
+        Disposable d = Disposables.empty();
+        observer.onSubscribe(d);
+
+        if (d.isDisposed()) {
+            return;
+        }
+        T value;
 
-        s.onSubscribe(EmptyDisposable.INSTANCE);
         try {
-            T v = callable.call();
-            if (v != null) {
-                s.onSuccess(v);
+            value = ObjectHelper.requireNonNull(callable.call(), "Callable returned null");
```

Comment:
You got the error message wrong.

[Travis CI](https://travis-ci.org/ReactiveX/RxJava/builds/307154950#L607)

Reply:
👍 fixed now :)